ipc/pipe/pipe_factory.go

```diff
@@ -31,12 +31,12 @@ type sr struct {
 	err error
 }
 
-// SpawnWorkerWithTimeout creates new Process and connects it to goridge relay,
+// SpawnWorker creates new Process and connects it to goridge relay,
 // method Wait() must be handled on level above.
-func (f *Factory) SpawnWorkerWithTimeout(ctx context.Context, cmd *exec.Cmd) (*worker.Process, error) {
+func (f *Factory) SpawnWorker(ctx context.Context, cmd *exec.Cmd, options ...worker.Options) (*worker.Process, error) {
```

Comment:
SpawnWorkerWithContext or just SpawnWorker?

Reply:
`SpawnWorkerWithContext`. This is a well-known naming in Go.